server/src/modules/apps/repository.ts

```diff
@@ -47,11 +47,14 @@ export class AppsRepository extends Repository<App> {
     });
   }
 
-  findById(id: string, organizationId: string, versionId?: string): Promise<App> {
+  findById(id: string, organizationId?: string, versionId?: string): Promise<App> {
```

Comment:
this was scoped under organizationId for some security fix I remember, can revert and use 
repositiry.findOne() directly if you need to get app by id only